UnityProject/Assets/Scripts/Items/Attributes.cs

```diff
@@ -176,8 +181,12 @@ public string Examine(Vector3 worldPos)
 
 	public RightClickableResult GenerateRightClickOptions()
 	{
-		return RightClickableResult.Create()
```

Comment:
it is more elegant to make ``AddElement`` return the object so you can chain as many ``AddElement`` as you need.
```cs
return RightClickableResults.Create()
    .AddElement("Examinable", OnExamine)
    .AddElement("PointTo", OnPointTo)
```